src/AccessibilityInsights.Rules/RunResult.cs

```diff
@@ -4,11 +4,30 @@
 
 namespace Axe.Windows.Rules
 {
+    /// <summary>
+    /// Contains the result of the evaluation by a rule of a specific element.
+    /// </summary>
     public class RunResult
     {
+        /// <summary>
+        /// Metadata about the rule that was run.
+        /// </summary>
         public RuleInfo RuleInfo{ get; set; }
```

Comment:
While you are here, do you mind adding a whitespace character between the RuleInfo property name and its opening bracket?

Reply:
Soitently